pcl_conversions/CMakeLists.txt

```diff
@@ -1,44 +1,63 @@
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.5)
 project(pcl_conversions)
 
-find_package(catkin REQUIRED COMPONENTS)
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
 
-find_package(PCL REQUIRED COMPONENTS common io)
+find_package(ament_cmake REQUIRED)
+find_package(Boost REQUIRED)
 find_package(Eigen3 REQUIRED)
+find_package(message_filters REQUIRED)
+find_package(PCL REQUIRED QUIET COMPONENTS common io)
+find_package(pcl_msgs REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(sensor_msgs REQUIRED)
+find_package(std_msgs REQUIRED)
 
-# There is a bug in the Ubuntu Artful (17.10) version of the VTK package,
-# where it includes /usr/include/*-linux-gnu/freetype2 in the include
-# directories (which doesn't exist).  This filters down to the PCL_INCLUDE_DIRS,
-# and causes downstream projects trying to use these libraries to fail to
-# configure properly.  Here we remove those bogus entries so that downstream
-# consumers of this package succeed.
-if(NOT "${PCL_INCLUDE_DIRS}" STREQUAL "")
-  foreach(item ${PCL_INCLUDE_DIRS})
-    string(REGEX MATCH "/usr/include/.*-linux-gnu/freetype2" item ${item})
-    if(item)
-      list(REMOVE_ITEM PCL_INCLUDE_DIRS ${item})
-    endif()
-  endforeach()
-endif()
 
-catkin_package(
-  INCLUDE_DIRS include
-  CATKIN_DEPENDS pcl_msgs roscpp sensor_msgs std_msgs
-  DEPENDS EIGEN3 PCL
+set(dependencies
+  pcl_msgs
+  rclcpp
+  sensor_msgs
+  std_msgs
+)
+
+set(includes
```

Comment:
I dont think you require this with the dependencies variable and the ament depends call

Reply:
Had to include the non-ROS package here (PCL) otherwise I got an error, couldn't find the header.

Maybe I'm missing something ? since I had the same issue for the `navsat_transform_node` in the `robot_localization` package.